posts/inside-rust/2019-10-15-compiler-team-meeting.md

```diff
@@ -28,9 +28,9 @@ Each week, we have general announcements from the team followed by check-ins fro
 
 ### [wg-learning](https://rust-lang.github.io/compiler-team/working-groups/learning/)
 
-`wg-learning` aims to make the compiler easier to learn by ensuring that rustc-guide and api docs are “complete”.
+`wg-learning` aims to make the compiler easier to learn by ensuring that rustc-dev-guide and api docs are “complete”.
```

Comment:
I don't know if that's the kind of modification that's allowed in the blog, but adding an active link to the guide would be convenient.

Reply:
addressed in 4367fad9d2bc955da039db41efc61627a0984c46